client/components/SignIn.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+
+export default (props) => {
+    let textInput
+    console.log('text', textInput);
+    return (
+      <div>
+        <h1>Sign In</h1>
+
+        <ul>
+          {props.teamMembers.map((teamMember) => {
+            return <a key={teamMember.id} href="#" onClick={() => {
+            props.signIn(teamMember.id)
+            }}><li> {teamMember.name}</li></a>
+          })}
+        </ul>
+
+        <form onSubmit= {e => {
+            e.preventDefault()
+            if (textInput.value.trim()) {
+              let name = textInput.value
+              props.signUp(name)
+          }}}>
```

Comment:
I don't think the browser will remember previously entered values, which is pretty nice, if it's not in a `<form>`, but I could be wrong.